project/components/resource_request/schemas.py

```diff
@@ -32,6 +32,8 @@ class ResourceRequestSchema(BaseSchema):
     username: str
     requested_for: str
     completed_at: datetime = None
+    message: str = None
+    vm_connections: dict = None
```

Comment:
`dict` is a parametric type and should not be used without params:
```suggestion
    vm_connections: dict[str, Any] | None = None
```

Reply:
Done